cmd/process.go

```diff
@@ -42,6 +42,27 @@ func Start(a *AppState) *cobra.Command {
 			logger := a.Logger
 			cfg := a.Config
 
+			flushInterval, err := cmd.Flags().GetDuration(flagFlushInterval)
+			if err != nil {
+				logger.Error("Invalid flush interval", "error", err)
+			}
+
+			flushOnly, err := cmd.Flags().GetBool(flagFlushOnlyMode)
+			if err != nil {
+				logger.Error("Invalid flush only flag", "error", err)
+				os.Exit(1)
```

Comment:
`os.Exit` can be avoided if the `cobra.Command` uses `RunE` instead of `Run`, see [Cobra docs](https://pkg.go.dev/github.com/spf13/cobra#Command) and search for `RunE`. then you can just return the error encountered instead of logging and aborting

Reply:
Oh cool! I'll check that out now